cmd/ossyscall.go

```diff
@@ -20,8 +20,17 @@ import (
 	"os"
 	"os/signal"
 	"syscall"
+
+	"github.com/dapr/cli/pkg/print"
 )
 
 func setupShutdownNotify(sigCh chan os.Signal) {
 	signal.Notify(sigCh, syscall.SIGTERM, syscall.SIGINT)
 }
+
+// createProcessGroupID creates a process group id for the current process.
```

Comment:
Since this is not a command, can we move these to `pkg/syscall/syscall.go` and `pkg/syscall/syscall_windows.go`?

Reply:
right, we can move. @mukundansundar any issue with this ?

Reply:
No issues